config/locales/in_person_proofing/fr.yml

```
@@ -37,8 +37,8 @@ fr:
       location:
         contact_info_heading: Information de contact
         distance:
-          one: 'à %{count} mile'
-          other: 'à %{count} milles'
+          one: '%{count} mile away'
+          other: '%{count} miles away'
```

I think we only want the unit to be in English, not the whole expression

```suggestion
          one: 'à %{count} mile'
          other: 'à %{count} miles'
```